lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +19,18 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+    text = text.lower()
+    text = text.replace("º", "")
```

Comment:
why?

Reply:
?

Reply:
why are you replacing this?

Reply:
this symbol is not a letter, so i decided to delete it